pkg/apis/kubeone/v1beta1/zz_generated.conversion.go

```diff
@@ -1401,6 +1401,7 @@ func autoConvert_kubeone_KubeOneCluster_To_v1beta1_KubeOneCluster(in *kubeone.Ku
 		out.DynamicWorkers = nil
 	}
 	out.MachineController = (*MachineControllerConfig)(unsafe.Pointer(in.MachineController))
+	// WARNING: in.OperatingSystemManager requires manual conversion: does not exist in peer-type
```

Comment:
I think we should define a manual conversion to set it to `false`. If I'm not mistaken, if we don't have this conversion, it would end up being `nil`, and then defaulted to `true`. I don't think we should enable OSM for v1beta1 users.

Reply:
It will only be defaulted for v1beta2 https://github.com/ahmedwaleedmalik/kubeone/blob/enable-osm-default/pkg/apis/kubeone/v1beta2/defaults.go#L223

Reply:
You're right, defaulting works only for the versioned APIs. Ignore this comment.